css/style.css

```diff
@@ -0,0 +1,358 @@
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+  font-family: sans-serif;
+  scroll-behavior: smooth;
+}
+
+body {
+  background-color: #233142;
+}
+
```

Comment:
- [ ] Please name the `index.css` instead of `style.css` as requested in the project requirements.